src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -560,17 +561,21 @@ def build_container_from_source_with_buildpack(self, image_name, source, cache_i
         except Exception as ex:
             raise ValidationError(f"Unable to run 'pack config' command to set default builder: {ex}") from ex
 
-        # If the user specifies a target port, pass it to the buildpack
-        if self.target_port:
-            command.extend(['--env', f"ORYX_RUNTIME_PORT={self.target_port}"])
-
         logger.warning("Selecting a compatible builder for the provided application source...")
         could_build_image = False
         for builder_image in builder_image_list:
             # Run 'pack build' to produce a runnable application image for the Container App
             # Specify the image as the 'build-cache' image to ensure that the local cache is used for build layers
             command = [pack_exec_path, 'build', cache_image_name, '--builder', builder_image, '--path', source, '--tag', image_name]
 
+            # If the user specifies a target port, pass it to the buildpack
+            if self.target_port:
+                command.extend(['--env', f"ORYX_RUNTIME_PORT={self.target_port}"])
+
+            # If the user has disabled telemetry collection for the CLI, propagate that to the builder
+            if not telemetry_core.is_telemetry_enabled():
+                command.extend(['--env', 'ORYX_DISABLE_TELEMETRY=true'])
```

Comment:
I assume that you tried this but just to be sure: could you please confirm that we can pass multiple `--env` values without problem?

Reply:
That's correct, you can provide multiple `--env` values to `pack build` -- we do this in a couple of our client tools and per their help documentation:

_This flag may be specified multiple times_

Reply:
Thanks for confirming!